source/standalone/workflows/robomimic/collect_demonstrations.py

```diff
@@ -13,7 +13,6 @@
 
 # add argparse arguments
 parser = argparse.ArgumentParser(description="Collect demonstrations for Isaac Lab environments.")
```

Comment:
This script already uses `--device` to select the teleoperation device. I believe it has to be renamed in order to avoid conflicts with the changes from this PR.

Reply:
Hi @AndrejOrsula, thanks a lot for the catch. Fix incoming very soon.